### PR TITLE
Whoa, first pull request

### PR DESCRIPTION
**Pull request by lekspyl-lf**
Please, approve of close this PR.

### CODE REVIEW

**Comment by northconstellation** [comment]
good idea

**Comment by ZubrytskyiAA** [comment]
+

**Comment by paull1311** [comment]
norm

File: README.md [hunk 1]
```diff
@@ -1 +1,4 @@
-Hello world!
\ No newline at end of file
+Hello world!
+I'm updated!
+
+Hey
```
Comment (lekspyl-lf) [inline README.md]: WTF is this?

**Comment by ganolmc** [comment]
OK

File: README.md (hunk 1 above)
Comment (lekspyl-lf) [reply README.md]: Oh, it's nothing

Comment (Ghyhulskyi) [reply README.md]: LOL

**Pull request merged**